src/main/java/Gumball/UI.java

```diff
@@ -0,0 +1,37 @@
+package Gumball;
+
+import java.util.Scanner;
+
+public class UI {
```

Comment:
Class name should be in PascalCase

```suggestion
public class Ui {
```